extensions/gc/GarbageCollector.js

```diff
@@ -132,6 +132,8 @@ class GarbageCollector extends EventEmitter {
             kafka: {
                 hosts: this._kafkaConfig.hosts,
                 site: this._kafkaConfig.site,
+                compressionType: this._kafkaConfig.compressionType,
+                requiredAcks: this._kafkaConfig.requiredAcks,
```

Comment:
won't this change the behavior on artesca?

If there are some good values, best to use them indeed;
but not sure if either
- we should rely on defaults from Kafka : maybe they are good (→ should be used as default everywhere, i.e. both s3c and zenko) or maybe we have better values (→ should be set in both products)
- we should actually have default values in backbeat, if both product don't agree
- if the products ahve different defaults, should probably be explicit : i.e. put the values in each product (or if we can define a "correct" default, but don't want to change it in S3C for compatibility: maybe defaut value in backbeat and "override" the default in s3c)

Reply:
The default values is what we use in Artesca, so no the behaviour won't change:
- Waiting for all brokers to ack the messages
- Using `Zstd` compression

I've talked with the S3C team and for RING 9.x we want to keep the same config from before :
- Only wait for one broker to ack the messages.
- Use broker level compression, no compression done by the producers.
The custom values will be set in the backbeat config in federation